webui/src/util/tablesUtil.jsx

```diff
@@ -1,29 +1,21 @@
 import React from "react";
+
 import {objects} from "../lib/api";
-import {useAPI} from "../lib/hooks/api";
-import {TableType, TreeItemType} from "../constants";
 
 /**
- * Checks whether a path is a table root and calculates its type.
+ * Checks whether a path is a delta table root.
  *
  * @param path the table path
  * @param repo the repo for the path
- * @param refs the ref in which the path is expected to be listed.
- * @return the type of table if the path is a table root, and null otherwise (for paths to objects or prefixes).
+ * @param ref the ref in which the path is expected to be listed.
+ * @return true if the path is a delat table root, false otherwise.
  */
-export function getTableType(path, repo, ref) {
-    const listResponse = useAPI(() => objects.list(repo.id, ref, path + "_delta_log/"), []);
-    if (listResponse.response !== null && listResponse.response.results.length !== 0) {
-        return TableType.DeltaLake;
+export async function isDeltaLakeTable(path, repo, ref) {
```

Comment:
This file has become a bit unnecessary - you can move this function to `tableRows.jsx`